packages/firestore/src/remote/remote_store.ts

```diff
@@ -134,14 +137,6 @@ export class RemoteStore {
   /** SyncEngine to notify of watch and write events. */
   syncEngine: RemoteSyncer;
 
-  /**
-   * Starts up the remote store, creating streams, restoring state from
-   * LocalStore, etc.
-   */
-  start(): Promise<void> {
```

Comment:
Okay, cool. Thanks for explaining.  I'd be down to have a no-op start() method, I think.  I like symmetry, especially since we have it in our other components.